x/storage/keeper/keeper.go

```diff
@@ -878,6 +914,11 @@ func (k Keeper) CopyObject(
 	}
 
 	if srcObjectInfo.PayloadSize == 0 {
+		lvg, err := k.virtualGroupKeeper.BindingEmptyObjectToGVG(ctx, dstBucketInfo.Id, dstBucketInfo.PrimarySpId, dstBucketInfo.GlobalVirtualGroupFamilyId)
+		if err != nil {
+			return sdkmath.ZeroUint(), err
+		}
+		objectInfo.LocalVirtualGroupId = lvg.Id
```

Comment:
Expose LVG or GVG id.

Reply:
the information about gvg or lvg of dst object is also not known when copy object

Reply:
if it is empty object, then it will get bindings, i think.